dictionaries/people-names/README.md

```diff
@@ -0,0 +1,89 @@
+# CSpell People Names Dictionary
+
+People Names dictionary for cspell.
+
+This is a pre-built dictionary for use with CSpell.
+
+This dictionary is an attempt at a collection of people names. It is not definitive. Contributions that add names are welcome.
+
+## Guidelines
+
+Due to the sensitive nature of names it is important to be clear about what is included in
+this dictionary. The idea here is to include names from all kinds of backgrounds and cultures, not just English or European (I call this out because this document was originally written in English including the examples below).
+
+- Each entry is part of a person's name, not the entire time.
+  For example, `John Smith` would not appear in the list, but `John` and `Smith` are ok.
+- No distinction is made between first, middle, last names. Each one can be an entry in the list.
+- All cultural and gender variants of a name are allowed.
+- A name should be common enough that 1000s of people have it.
+- A name should not be a slur, disparaging, or derogatory term.
+  For example, `Shithead` is not allowed.
+
+  Exceptions:
+  Names that were once common, but have become slang
+
+  - `Dick` is allowed because it is a common derivate of `Richard`.
+  - `Fanny` is allowed because it is a common derivate of `Frances`.
+
+### Adding Names
+
+Please open a pull to add names to the `src/names.txt` file.
+
+- Add new names to the end of the file.
+- One name per line.
+
+### Name Removal
+
+Names can be removed if they are:
+
+- Are actually misspelled. (`Johhn` would be replaced with `John`)
+- Are not a person's name.
```

Comment:
This one would be difficult to check, people would argue

You should add something as another point

- no matter the rules listed above project maintainers might refuse things

Reply:
- Moderators reserve the right to refuse changes and additions.

Reply:
Maintainers sounds less negative. We don't have moderators